.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "node": true,
+        "es6": true,
+        "jest": true
+    },
+    "extends": [ "eslint:recommended" ],
+    "rules": {
+        "indent": [
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
```

Comment:
I wanted to do this but I saw many places that they were like this. I don't know why. It is ugly this way but there should be some reason behind it. I will search for it more.

Maybe this way it is more similar to yaml format! Just a guess.

Reply:
@ahangarha Nah, trust me it's just a code style preference. Not everything should have a hidden meaning.

Reply:
@ahangarha Take a look: https://eslint.org/docs/user-guide/configuring/configuration-files#using-eslintrecommended

![image](https://user-images.githubusercontent.com/42304709/153493046-cffed2cc-7a1c-4a32-bf52-dcfe04f9879c.png)

We should be creative sometimes :3

Reply:
Ok. I will make it one line. But I think they wrote it in multiple line to keep it similar to yaml file which makes sense since many config files are written in that format. But I will make it one line.